src/starbase/config.ts

```diff
@@ -18,6 +20,42 @@ const integrationSchema = {
   additionalProperties: true,
 };
 
+function integrationConfigToEnvFormat(
+  config: StarbaseIntegration['config']
+) {
+  let envFileContents = '';
+
+  for (const configEntryName in config) {
+    envFileContents += `${configEntryName}=${config[configEntryName]}\n`;
+  }
+
+  return envFileContents;
+}
+
+/**
+ * This is the `.env` file that is written to the root of the Starbase project.
+ */
+async function writeNeo4jRootConfig(storage: StarbaseStorage) {
+  await fs.writeFile(
+    '.env',
+`NEO4J_URI=${storage.config.uri}
+NEO4J_USER=${storage.config.username}
+NEO4J_PASSWORD=${storage.config.password}
```

Comment:
The `.env` file written out now is formatted cleanly. Previously, there was whitespace at the beginning of every line.